src/services/authorization.test.ts

```diff
@@ -0,0 +1,2660 @@
+import { ItemTagType, PermissionLevel, UUID } from '@graasp/sdk';
```

Comment:
Very nice all those unit tests :sparkles: for readability purposes, i would suggest adding blank newlines between `it` blocks as well as `describe` blocks. It makes it easier to distinguish the test cases visually.